app/di.php

```diff
@@ -9,10 +9,13 @@
 // Blocks
 $registrar->addInstance(new \GovUKBlogs\Blocks\BlockCategory());
 $registrar->addInstance(new \GovUKBlogs\Blocks\AllowedBlocks());
+// $registrar->addInstance(new \GovUKBlogs\Blocks\Button\Block());
 $registrar->addInstance(new \GovUKBlogs\Blocks\Details\Block());
 $registrar->addInstance(new \GovUKBlogs\Blocks\InsetText\Block());
 $registrar->addInstance(new \GovUKBlogs\Blocks\Accordion\Block());
 $registrar->addInstance(new \GovUKBlogs\Blocks\AccordionRow\Block());
+$registrar->addInstance(new \GovUKBlogs\Blocks\StyleVariations\Scripts());
+$registrar->addInstance(new \GovUKBlogs\Blocks\Variations\Scripts());
```

Comment:
Though I understand the logic in this structure, I think having `StyleVariations` and `Variations` as separate locations brings potential confusion to the codebase, for those unclear on the distinction. Especially, considering the inner files have the same names.

I'd be inclined to maintain consistency with how we're managing block JS in the theme already i.e. locating within `assets/js`, as per `button.js` and `accordion.js`. The pre-existing `Scripts.php` can then be used to load the assets.

Reply:
I guess I put them there because they influence the block editor rather than being frontend focused - but I can see why we might want them in assets - I have moved them over now